.github/workflows/build-all.yml

```diff
@@ -52,10 +53,10 @@ jobs:
           mkdir C:/handsontable
           git clone -b $env:GITHUB_REF_NAME https://github.com/handsontable/handsontable C:/handsontable --depth 1
 
-      - name: Use Node.js ${{ env.NODE_VERSION }}
+      - name: Use Node.js
         uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # https://github.com/actions/setup-node/releases/tag/v4.0.2
         with:
-          node-version-file: '.nvmrc'
```

Comment:
maybe would be enough to add  ` working-directory: C:/handsontable` as it's not able to find the file

Reply:
I tried without success